Update New-TransportRule.md (#9204)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/New-TransportRule.md

- OOF: Auto-reply messages configured by the user.
- AutoForward: Messages automatically forwarded to an alternative recipient.

> - Encrypted: Encrypted messages. In thin clients like OWA (Outlook Web App) encryption as a message type is currently not supported.

Reviewer: ```suggestion
- Encrypted: Encrypted messages. In thin clients like Outlook Web App (OWA), encryption as a message type is currently not supported.
```

> - Encrypted: Encrypted messages. In thin clients like Outlook Web App (OWA), encryption as a message type is currently not supported.

Reviewer: ```suggestion
- Encrypted: Encrypted messages. In thin clients like Outlook on the web, encryption as a message type is currently not supported.
```
